types/react-native/index.d.ts

```diff
@@ -1545,7 +1546,8 @@ export interface ViewStyle extends FlexStyle, TransformsStyle {
     borderTopColor?: string;
     borderTopLeftRadius?: number;
     borderTopRightRadius?: number;
-    borderTopWidth?: number
+    borderTopWidth?: number;
+    color?: string;    
     opacity?: number;
```

Comment:
`@types/react-native` have the correct interface definition, just ignore what this pr said before 😅 

The problem is my project  declared the wrong type  https://github.com/ant-design/ant-design-mobile/blob/master/components/search-bar/style/index.tsx#L22.

Sorry for bother you guys.